pimd/pim_msg.c

```diff
@@ -97,6 +97,68 @@ uint8_t *pim_msg_addr_encode_ipv4_source(uint8_t *buf, struct in_addr addr,
 	return buf + PIM_ENCODED_IPV4_SOURCE_SIZE;
 }
 
+uint8_t *pim_msg_addr_encode_ipv6_source(uint8_t *buf, struct in6_addr addr,
```

Comment:
Need to move the below functions under #if PIM_IPV == 6

Reply:
Nope, they work regardless of which protocol we're compiling for. We include IPv6 addrs in the secondary address list in IPv4 too…